Exiled.API/Features/Roles/OverwatchRole.cs

```diff
@@ -23,9 +23,22 @@ public class OverwatchRole : SpectatorRole
         internal OverwatchRole(OverwatchGameRole baseRole)
             : base(baseRole)
         {
+            Internal = baseRole;
         }
 
         /// <inheritdoc/>
         public override RoleTypeId Type => RoleTypeId.Overwatch;
+
+        /// <summary>
+        /// Gets the game <see cref="OverwatchGameRole"/>.
+        /// </summary>
+        protected OverwatchGameRole Internal { get; }
```

Comment:
This could just overwrite the "Base" property with ``public new OverwatchGameRole Base { get; }`` instead of introducing a second property for the same value.

Reply:
done